src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Features/RegisterRenewConfirm.cs

```diff
@@ -73,5 +73,64 @@ public async Task Test()
                     change.CourseName,
                 });
         }
+
+
+        [Test]
+        public async Task ConfirmingCommitmentStatementConcurrentToApprovedChangeConfirmsCorrectStatement()
+        {
```

Comment:
Location of this class, is a worry, as it's hidden in the Features folder but it isn't a feature and because we have mingled in the steps files with the feature files, it's not obvious this is a different way of testing.

Reply:
I have moved it to a "WorkflowTests" folder.